### DISCUSSION TITLE
Europe/Latvia VoLTE works with 3 major carriers: Bite, LMT, Tele2

### DISCUSSION THREAD

**Discussion by vv371** [kyujin-cho/pixel-volte-patch#174] [Carrier-specific Experiences]
Hello, @kyujin-cho thanks for the great tool, please update the table for Latvia, tested on Pixel 7, VoLTE works flawlessly. was considering selling the phone )

**Comment by silbaums** [comment]
yes works on oneplus and sony xperia, where volte isn't enabled by OEM, both Bite and Tele2 huge thanks @kyujin-cho
